### PR TITLE
ClustalW: include additional options

### PR DESCRIPTION
**Pull request by gallardoalba**

### CODE REVIEW

**Review by nsoranzo — COMMENTED** [review]
Can you please remove `tools/clustalw/repository_dependencies.xml` ?

By the way, we could maybe move this tool to the tools-iuc repository after we merge this?

File: tools/clustalw/rgClustalw.xml
Comment (gallardoalba) [reply tools/clustalw/rgClustalw.xml:51]: Sure!

File: tools/clustalw/rgClustalw.xml
Comment (gallardoalba) [reply tools/clustalw/rgClustalw.xml:122]: Well, I just changed it because didn't seem very intuitive, since it allows aligning nucleic acids or proteins, but I'll undo the change.

**Comment by gallardoalba** [comment]
I think all changes have been included, thanks for the review!

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
Thanks @gallardoalba ! Let me know if you need help with the migration to tools-iuc.

**Comment by gallardoalba** [comment]
> Thanks @gallardoalba ! Let me know if you need help with the migration to tools-iuc.

I guess that it requires copying the folder and modifying the .shed file, is it necessary any additional step?

**Comment by nsoranzo** [comment]
> > Thanks @gallardoalba ! Let me know if you need help with the migration to tools-iuc.
> 
> I guess that it requires copying the folder and modifying the .shed file, is it necessary any additional step?

Yep, only the `remote_repository_url` not the `owner`.
Then, before merging into tools-iuc, we need to modify the permissions on the ToolShed repo to allow write access by iuc.
Lastly, we need another PR to remove clustalw from this repo.

`column_maker` is the last tool we migrated, I think, if you'd like to check how that was done.

**Pull request merged**
